tasks/extract-binaries-from-image/tests/test-extract-binaries-from-image.yaml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-extract-binaries-from-image
+spec:
+  description: |
+    Run the extract-binaries-from-image task
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-snapshot
+            image: quay.io/hacbs-release/release-utils:48a1cd6c939eda25382472135be4afe2c9b4b703
```

Comment:
This should be updated to the redhat-appstudio ref. See any of the other tasks for an example. It was changed recently

Reply:
Done!